task/sql-tasks.js

```diff
@@ -247,7 +400,19 @@ async function task_1_18(db) {
  *
  */
 async function task_1_19(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            Orders.CustomerID as "CustomerID",
+            Customers.CompanyName as "CompanyName",
+            SUM(OrderDetails.UnitPrice * OrderDetails.Quantity) AS "TotalOrdersAmount, $"
+        FROM Orders INNER JOIN OrderDetails INNER JOIN Customers
+            ON Orders.OrderID = OrderDetails.OrderID 
+            AND Orders.CustomerID = Customers.CustomerID
+        GROUP BY Orders.CustomerID
+        HAVING SUM(OrderDetails.UnitPrice * OrderDetails.Quantity) > 10000
+        ORDER BY SUM(OrderDetails.UnitPrice * OrderDetails.Quantity) DESC, CustomerID ASC

```

Comment:
Please, simplify ORDER BY and HAVING clauses using aliases from the prevuous step (don't forget escape quotes in sql query in node code)